FirebaseDynamicLinks/Sources/FIRDynamicLinkNetworking.m

```diff
@@ -116,33 +117,66 @@ - (void)resolveShortLink:(NSURL *)url
     @"sdk_version" : FDLSDKVersion
   };
 
-  FIRNetworkRequestCompletionHandler resolveLinkCallback = ^(NSData *data, NSError *error) {
```

Comment:
Whenever a call to `reopenAttribution` endpoint took place response object was disregarded despite the fact that its status code indicated an error. On the other hand, `error` object itself was not populated and was `nil`. As a result, all traces of an issue used to be lost here and the calling code had no clue that the url was not allowed.

<img width="860" alt="Screenshot 2020-04-29 at 17 58 06" src="https://user-images.githubusercontent.com/66690260/85116514-1dff6680-b226-11ea-9c8c-703fec668153.png">